hash/ibm5170.xml

```diff
@@ -590,221 +590,106 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<software name="freedos13rc1">
-		<description>FreeDOS 1.3 Release Candidate 1 (bootdisk)</description>
-		<year>2019</year>
+	<software name="freedos13">
```

Comment:
The bootdisk is tied with the CDROM release, should be moved in `ibm5170_cdrom.xml`

Reply:
> The bootdisk is tied with the CDROM release, should be moved in `ibm5170_cdrom.xml`

Agreed :) This was one of my earlier PRs, I'll fix this up now.